src/data/complex/basic.lean

```diff
@@ -226,7 +226,7 @@ lemma eq_conj_iff_im {z : ℂ} : conj z = z ↔ z.im = 0 :=
 ⟨λ h, add_self_eq_zero.mp (neg_eq_iff_add_eq_zero.mp (congr_arg im h)),
   λ h, ext rfl (neg_eq_iff_add_eq_zero.mpr (add_self_eq_zero.mpr h))⟩
 
-@[simp] lemma star_def : (has_star.star : ℂ → ℂ) = conj := rfl
+lemma star_def : (has_star.star : ℂ → ℂ) = conj := rfl
```

Comment:
This seems like it might be quite a significant change to the simp-set - can you mention that this PR includes this change in the PR description?

Reply:
You could also `attribute [-simp]` it in the file you introduce the `is_R_or_C` version.

Reply:
Actually neither file imports the other one. I think the linter complains because in `all.lean` the `is_R_or_C` version comes first but it's quite artificial.